connector.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def get_ephemeral(service, project_name, instance_name, pub_key):
+    """
+    A helper function that requests an ephemeral certificate from the
+    Cloud SQL Instance.
+
+    Args:
+        service: A service object created from the Google Python API client
+            library. Must be using the SQL Admin API.
+        project (String): A string representing the name of the project.
+        instance (String): A string representing the name of the instance.
+            Usually found in environment variable 'CLOUD_SQL_INSTANCE_NAME.'
+        pub_key (String): A string representing PEM-encoded RSA public key.
+
+    Returns:
+        An ephemeral certificate from the Cloud SQL instance that allows
+        authorized connections to the instance.
+
+    Raises:
+        TypeError: If one of the arguments passed in is None.
+    """
+
+    if (
+        service is None or
+        project_name is None or
+        instance_name is None or
+        pub_key is None
+    ):
+        print("Cannot take None as an argument.")
+        raise(TypeError)
```

Comment:
I think you can take care of both lines at once by doing:

```suggestion
        raise TypeError("Cannot take None as an argument")
```